web/client/epics/identify.js

```diff
@@ -198,10 +200,40 @@ export default {
             }
             return disableAlwaysOn || !stopFeatureInfo(store.getState() || {});
         })
-            .switchMap(({point, layer}) => Rx.Observable.of(featureInfoClick(point, layer))
-                .merge(Rx.Observable.of(addPopup(uuid(), { component: IDENTIFY_POPUP, maxWidth: 600, position: {  coordinates: point ? point.rawPos : []}}))
-                    .filter(() => isMapPopup(store.getState()))
-                )),
+            .switchMap(({point, layer}) => {
+                // calculate a query for edit
+                const lng = get(point, 'latlng.lng');
+                const lat = get(point, 'latlng.lat');
+                const hook = getHook(GET_COORDINATES_FROM_PIXEL_HOOK);
+                const radius = calculateCircleRadiusFromPixel(
+                    hook,
+                    projectionSelector(store.getState()),
+                    get(point, 'pixel'),
+                    get(point, 'latlng'),
+                    10
+                );
+
+                return Rx.Observable.of(setEditFeatureQuery({
```

Comment:
Can you add this to the point object to the feature info click, so it can be reused later without handling a separate state ?